src/__tests__/e2e.test.js

```diff
@@ -38,8 +38,7 @@ describe('e2e', () => {
   it('accepts a dispatch function in options', () => {
     @track(testDataContext, { dispatch })
     class TestOptions extends React.Component {
-      @track(testData)
-      blah = () => {}
+      @track(testData) blah = () => {};
```

Comment:
Eek! Does prettier really in-line these? Do you know if there's any way around that? Not a huge deal, but just curious.

Reply:
I'll have a look. I think it is treating them like bracketless conditions eg. `if (true) a = b` kind of thing. I'll have a look if it's prettier or eslint that's enforcing this.

Reply:
Looks like there is an [issue](https://github.com/prettier/prettier/issues/1974) up for discussion in prettier for this. If there was multiple decorators then it would line break but lines applied to variables reduce to one line but not functions or methods. 

eg. 

```
class a {
    @track() somevar = () => {}

    @track()
    somevar() {
    }
}


```